DQM/Integration/python/clients/pixel_dqm_sourceclient-live_For_Timing_Scan_cfg.py

```diff
@@ -0,0 +1,182 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
The file name should be pixel_timing_scan_dqm_sourceclient-live_cfg.py or similar. No touching anything after "dqm_".